noodles-editor/src/noodles/components/block-library.module.css

```diff
@@ -162,6 +162,14 @@
   box-shadow: 0 4px 12px rgba(0, 0, 0, 0.3);
 }
 
+/* Disable hover effects in keyboard mode */
+.blockLibraryContent[data-input-mode='keyboard'] .blockLibraryCard:hover {
```

Comment:
How about a `:not()` on the hover class instead, or affirmatively testing for the mouse mode just where we want hover